ACL config doc, updated OIDC config (#99)
questdb/documentation: documentation/configuration.md

<ConfigTable rows={replicationConfig} />

### Identity and Access Management (IAM)

:::note

> Identity and Access Management is [Enterprise](/enterprise/) and [Cloud](/cloud/) only.

Reviewer: ```suggestion
Identity and Access Management is available within [QuestDB Enterprise](/enterprise/).
```
